tox.ini

```
@@ -1,17 +1,26 @@
-# Tox (http://tox.testrun.org/) is a tool for running tests
+# Tox (https://tox.wiki/) is a tool for running tests
 # in multiple virtualenvs. This configuration file will run the
 # test suite on all supported python versions. To use it, "pip install tox"
 # and then run "tox" from this directory.
 
 [tox]
-# pip is broken on py32
-envlist = py36, py37, py38, py39, py310, py311, py312, py313
+env_list = py36, py37, py38, py39, py310, py311, py312, py313
```

More dropping of py36 and py37:
```suggestion
env_list = py38, py39, py310, py311, py312, py313
```